shelley/design-spec/delegation_design_spec.tex

```diff
@@ -545,276 +573,197 @@ \section{Design of Delegation}
-addresses have stake rights corresponding to the funds
-at the address. To exercise these stake rights, and to receive staking
-rewards, each address must be associated with a \emph{registered} stake
-key. This involves registering a stake key on the chain and using
-addresses that refer to this stake key. Each registered stake key has a
-corresponding \emph{reward account} which is used to collect and claim
-staking rewards. To exercise stake rights associated with a registered
-stake key, those rights must be \emph{delegated} to a registered stake pool.
-This involves posting a delegation certificate on the chain identifying
-the chosen stake pool. Registered stake pools participate in the Proof
-of Stake protocol using the key for their stake pool to sign blocks.
-Only registered stake pools participate in the PoS protocol, but anyone
-can register a private stake pool for ``self staking''. For each stake
-pool, the set of stake keys that delegate to it are known, and staking
-rewards can be paid into the reward accounts associated with each stake
-key. Rewards can accumulate in reward accounts over multiple epochs and
-can be reclaimed as part of a transaction using a special input type.
-
-Thus, the overall structure is that addresses with stake rights are
-associated with a stake key, and the stake key delegates to a stake
-pool, and the stake pool takes part in the PoS protocol. Private staking
-works in exactly the same way, using a stake key, and delegating, but
-using a private stake pool.
-
-Figure~\ref{fig:overview-delegation} shows some of the concepts involved in
-delegation, and their relationships. The light-blue nodes represent concepts, which
-do not have a representation in the system, whereas light-brown nodes represent
-system entities.
-
-\begin{figure}[ht]
-  \centering
-  \begin{tikzpicture}[ align = center
-                     , node distance = 8em and 10em
-                     , text width = 5em
-                     , font = \footnotesize
-                     , >={Latex[width=0.5em, length=0.5em]}
-                     , every node/.style = { rectangle
-                                         , rounded corners
-                                         , draw = black
-                                         , align = center
-                                         , minimum height = 4em }
-                     ]
-
+Delegation is a separation of the control over the movements of funds and the
+rights (and obligations) in the PoS protocol that are associated with those
+funds. We achieve this separation by modelling it in the address structure. We
+will distinguish between \emph{value addresses} that determine how funds can be
+spent, and \emph{stake addresses} that define if and how those funds take part
+in the PoS protocol. Actual coins will belong to an (overall) \emph{address},
+consisting of both a value address and a stake address.
```

Comment:
I think this is not how we currently describe it to ourselves and not the clearest way to think about it.

As in the dot diagram, there are only two kinds of addresses stake addresses and normal addresses for which we should pick a standard term. I've been using payment address recently, but value address is ok. We originally just said "address" but that was before we introduced stake addresses, and then "address" vs "stake address" feels unclear, whereas "payment address" vs "stake address" is much clearer.

So I think we should describe it thus:

> We distinguish between \emph{payment addresses} that determine how funds can be spent, and \emph{stake addresses} that define if and how the stake rights of those funds take part in the PoS protocol. Coins belong to payment addresses. Each payment address (optionally) referrs to a stake address. This delegates the stake rights of any funds held at the payment address to the corresponding stake address. The stake address delegates to a stake pool that participates directly in the PoS protocol. Thus overall there are two steps to the delegation of stake rights: a payment address refers to a stake address; and the stake address delegates to a stake pool.

Reply:
Oh right! I should have gone over the introduction again after finishing the chapter on address structure.